Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+                                                    <add place="overstrike">
+                                                        ለገብርኪ፡ 
+                                                        <persName ref="PRS14555EdaKrestos"><del rend="effaced"/>እ<gap reason="illegible"/>ክርስቶስ፡</persName>
+                                                        <add place="above"><persName ref="PRS14557WaldaMikael">ወልደ፡ ሚካኤል</persName></add>
+                                                        ለዓለመ፡ ዓለም፡ አሜን።
+                                                    </add>
+                                            </subst>
+                                ወኢትርሐቂ፡ ወትረ፡ እምኔሆሙ<hi rend="partialRubric">፨</hi> 
+                                በኵሉ፡ መዋዕሊሆሙ<hi rend="partialRubric">፨</hi> 
+                                ዘምስለ፡ ደቂቆሙ፡ ወአዋልዲሆሙ<hi rend="partialRubric">፨</hi> 
+                                ይእዜኒ<hi rend="partialRubric">፨</hi> 
+                                ወዘልፈኒ<hi rend="partialRubric">፨</hi> 
+                                ወለዓለመ፡ ዓለም<hi rend="partialRubric">፨</hi> 
+                                ኣሜን<hi rend="partialRubric">፨</hi> 
+                                ወኣሜን<hi rend="partialRubric">፨</hi> 
+                            </explicit>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="1ra" to="30va"/>
+                                <title type="complete" ref="LIT1146Argano#Monday"></title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39</citedRange></bibl>
+                                    notes that folios are in the wrong order, correcting it to: 
+                                    <locus target="#1"/>, <locus target="#3"/>, <locus target="#4"/>, 
+                                    <locus target="#7"/>, <locus target="#16"/>, <locus target="#5"/>, 
+                                    <locus target="#6"/>, <locus target="#2"/>, <locus from="8" to="15"/>,
+                                    <locus from="17" to="30"/>.
+                                </note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="30vb" to="55va"/>
+                                <title type="complete" ref="LIT1146Argano#Tuesday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="55va" to="79vb"/>
+                                <title type="complete" ref="LIT1146Argano#Wednesday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="80ra" to="86rb"/>
+                                <locus from="87ra" to="104vb"/>
```

Comment:
why do you have two locus?

Reply:
I assume that Jonas wanted to document that there are two additions (a1 and a2) in between on folio 86v. I think this is correct. At least, this is the way I do it in similar cases.